### PR TITLE
chore: Remove suggestion to use Bitcode

### PR DESCRIPTION
**Pull request by brustolin**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes
Apple don't support Bitcode anymore, so we remove the suggestions to use Bitcode to extract symbols.

Related to https://github.com/getsentry/sentry-docs/issues/6756

### CODE REVIEW

File: src/wizard/react-native/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ npx @sentry/wizard -s -i reactNative
 
 [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [setup manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer.
 
-- iOS Specifics: When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps. However, if you are using bitcode, you will need to disable the “Upload Debug Symbols to Sentry” build phase and then separately upload debug symbols from iTunes Connect to Sentry.
+- iOS Specifics: When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps. However.
```
Comment (bitsandfoxes) [inline src/wizard/react-native/index.md:20]: I think the `However` got left behind.

**Review by kahest — COMMENTED** [review]
Thanks for pushing this @brustolin!

File: src/wizard/react-native/index.md (hunk 1 above)
Comment (kahest) [reply src/wizard/react-native/index.md:20]: `m`: Does the user need to actively hook this up, or do we patch this? @krystofwoldrich

Comment (brustolin) [reply src/wizard/react-native/index.md:20]:
```suggestion
- iOS Specifics: When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps. ```

File: src/platform-includes/debug-symbols-apple/_default.mdx [hunk 2]
```diff
@@ -168,23 +168,3 @@ If your App Store Connect API key is revoked on the Apple side, the Sentry custo
 ![Warning to update credentials in Custom Repositories](custom-repositories-warning.png)
 
 When using Fastlane's _upload_to_testflight_ action, it's recommended to pass the _skip_waiting_for_build_processing_ parameter to speed up the action.
-
-## Bitcode {#dsym-with-bitcode}
```
[the remark this answers is not in the document]
Comment (brustolin) [reply src/platform-includes/debug-symbols-apple/_default.mdx:172]: I believe Is not possible to retrieve the symbols from the appconnect anymore.

**Comment by brustolin** [comment]
@bitsandfoxes

Since Im updating docs on `bitcode` i will update for Unity as well. 
Please check if my changes make sense.

**Comment by brustolin** [comment]
> There's two more places we can clean up:

Unity troubleshooting already mention that bitcode is deprecated, so I did not want to remove it for now. I guess this would be another source of information for old projects that started to have problems because of bitcode.

**Comment by kahest** [comment]
> > There's two more places we can clean up:
> 
> Unity troubleshooting already mention that bitcode is deprecated, so I did not want to remove it for now. I guess this would be another source of information for old projects that started to have problems because of bitcode.

I think we can/should keep the mention, but consider the wording

@bitsandfoxes is the statement `By default, Unity exports an Xcode project with `Bitcode` enabled. This feature is [deprecated in Xcode 14](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes) and disables the automatic symbol upload.` still correct? Any more information we can give to users?

**Review by shanamatthews — APPROVED** [review]
No issues from docs side!

**Review by bitsandfoxes — APPROVED** [review]
Sorry for the delay! Had to update and release the Unity SDK to make it up to date with our shiny new docs! 🎉

**Pull request merged**